v-next/hardhat/src/internal/cli/telemetry/analytics/analytics-subprocess.ts

```diff
@@ -0,0 +1,17 @@
+import { postJsonRequest } from "@ignored/hardhat-vnext-utils/request";
+
+// These keys are expected to be public
+const ANALYTICS_URL = "https://www.google-analytics.com/mp/collect";
+const API_SECRET = "fQ5joCsDRTOp55wX8a2cVw";
+const MEASUREMENT_ID = "G-8LQ007N2QJ";
+
+(async () => {
+  const payload = JSON.parse(process.argv[2]);
+
+  await postJsonRequest(ANALYTICS_URL, payload, {
+    queryParams: {
+      api_secret: API_SECRET,
+      measurement_id: MEASUREMENT_ID,
+    },
+  });
+})().catch((_err: unknown) => {});
```

Comment:
the lambda is not needed, there's top-level-await support in node when you use ESM, which we do in v3

Reply:
[Done here](https://github.com/NomicFoundation/hardhat/pull/5545/commits/f1ab5a47cabf4d13d1ff3419a96b66e3fec4e2d8)